FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1320,11 +1332,117 @@ public function testMultiByteCharactersAreExportedInLowercase($articleConfigurat
         $cat_urls = $values['cat_url']->getValues();
 
         $this->assertEquals(
-            $cat_urls,
             [
-                '/genusswelten/',
-                '/oel/'
+                '/genusswelten',
+                '/oel'
+            ],
+            $cat_urls
+        );
+    }
+
+    public function categoriesWithSlashProvider()
+    {
+        try {
+            /** @var ArticleResource $resource */
+            $resource = Manager::getResource('Category');
+
+            $resource->create([
+                'id' => 2000,
+                'name' => 'PC / Parts',
+                'parent' => '3',
+            ]);
+            $resource->create([
+                'id' => 3000,
+                'name' => 'Processor/Mainboards',
+                'parent' => '2000'
+            ]);
+        } catch (Exception $e) {
+            echo sprintf('Exception: %s', $e->getMessage());
+        }
+
+        $articleConfiguration = [
+            'name' => 'FindologicArticle 1',
+            'active' => true,
+            'tax' => 19,
+            'supplier' => 'Findologic',
+            'categories' => [
+                ['id' => 3],
+                ['id' => 5],
+                ['id' => 3000]
+            ],
+            'images' => [
+                ['link' => 'https://via.placeholder.com/300/F00/fff.png'],
+                ['link' => 'https://via.placeholder.com/300/09f/000.png'],
+            ],
+            'mainDetail' => [
+                'number' => 'FINDOLOGIC1',
+                'active' => true,
+                'inStock' => 16,
+                'prices' => [
+                    [
+                        'customerGroupKey' => 'EK',
+                        'price' => 99.34,
+                    ],
+                ]
+            ],
+            'filterGroupId' => 1,
+        ];
+
+        return [
+            'Category is exported correctly' => [
+                'articleConfiguration' => $articleConfiguration,
             ]
+        ];
+    }
+
+    /**
+     * @dataProvider categoriesWithSlashProvider
+     *
+     * @param array $articleConfiguration
+     *
+     * @throws Exception
+     */
+    public function testCategoryNamesWithSlashesAreExportedCorrectly($articleConfiguration)
+    {
+        $articleFromConfiguration = $this->createTestProduct($articleConfiguration);
```

Comment:
Use `$this->markTestSkipped()` with an appropriate message, in case it is Shopware 5.2. Use the environment for that, we already use it in other places in the tests.